packages/lambda-xyz/src/tile-request.ts

```diff
@@ -112,3 +116,33 @@ export default async (info: ReqInfo): Promise<LambdaHttpResponseAlb> => {
     response.buffer(res.buffer, 'image/png');
     return response;
 };
+
+const wmts = (info: ReqInfo, wmtsData: TileDataWmts): LambdaHttpResponseAlb => {
+    const response = new LambdaHttpResponseAlb(200, 'ok');
+
+    const host = info.getHeader('host') || '';
```

Comment:
I do not believe this will give us the right value, we also need to do `{a-f}.tiles..`

I think we may have to pass this in as a environment variable which i think it may already be.

Reply:
Leaving off the host for now (url is `/v1/tiles/...`)